x-pack/plugins/cases/common/api/connectors/get_connectors.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import * as rt from 'io-ts';
+import { CaseConnectorRt } from './connector';
+import { CaseExternalServiceBasicRt } from '../cases';
+
+const CaseConnectorPushInfoRt = rt.intersection([
+  rt.type({
+    needsToBePushed: rt.boolean,
+    hasBeenPushed: rt.boolean,
+  }),
+  rt.partial(
+    rt.type({
+      latestUserActionPushDate: rt.string,
```

Comment:
Updated these from `latest/oldest PushDate` to `latest/oldestUserActionPushDate` to make it clear where the date is coming from because it will be slightly newer than the push payload timestamp.